### PR TITLE
Allow configuring array child label

### PR DESCRIPTION
**Pull request by iluuu1994**

### CODE REVIEW

File: packages/material/test/renderers/MaterialArrayLayout.test.tsx [hunk 1]
```diff
@@ -431,4 +441,54 @@ describe('Material array layout', () => {
...
+          <JsonFormsReduxContext>
+            <MaterialArrayLayout
+                schema={schema}
+                uischema={uischemaWithSortOption}
+            />
+          </JsonFormsReduxContext>
+        </Provider>
+    );
+    const childLabel1 = wrapper
+        .find('.MuiGrid-root.MuiGrid-item.MuiGrid-grid-xs-10.MuiGrid-grid-md-11')
```
Comment (iluuu1994) [inline packages/material/test/renderers/MaterialArrayLayout.test.tsx]: Should we add a class to that element so it can be selected easier?

**Comment by sdirix** [comment]
Thank you very much for the contribution. Much appreciated! We'll take a closer look at it soon.

File: packages/material/test/renderers/MaterialArrayLayout.test.tsx (hunk 1 above)
Comment (sdirix) [reply packages/material/test/renderers/MaterialArrayLayout.test.tsx]: This question made me realize that we have no proper semantics defined for this label. I now introduced `aria-labelledby` to add respective semantics.

**Comment by iluuu1994** [comment]
Looks good, thanks!

**Comment by sdirix** [comment]
Thanks again for the contribution!

**Comment by iluuu1994** [comment]
Cool, thank you @sdirix! 😊

**Pull request merged**